package.json

```diff
@@ -5,6 +5,7 @@
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
+    "eep": "^0.3.1",
```

Comment:
Do we still need this library?

Reply:
No we don't. My bad, forgot to delete it

Reply:
You can remove it by running something like `npm uninstall eep` or `sudo npm uninstall eep`